node/modules/lp2p/rcmgr.go

```diff
@@ -28,8 +28,8 @@ import (
 func ResourceManager(connMgrHi uint) func(lc fx.Lifecycle, repo repo.LockedRepo) (network.ResourceManager, error) {
 	return func(lc fx.Lifecycle, repo repo.LockedRepo) (network.ResourceManager, error) {
 		envvar := os.Getenv("LOTUS_RCMGR")
-		if envvar == "" || envvar == "0" {
-			// TODO opt-in for now -- flip this to enabled by default once we are comfortable with testing
+		if envvar == "0" {
```

Comment:
We probably want some more testing before defaulting to enabled

Reply:
sure, its master though

Reply:
dropped, will make follow up pr wiht it.